R/loo_difference_plot.R

```diff
@@ -0,0 +1,151 @@
+#' Compare models across domains
+#' 
+#' The LOO difference plot shows how the ELPD of two different models
+#' changes when a predictor is varied. This can is useful for identifying
+#' opportunities for model stacking or expansion.
+#' 
+#' @param y A vector of observations. See Details.
+#' @param psis_object_1,psis_object_2 If using loo version 2.0.0 or greater, 
+#' an object returned by the `[loo::psis()]` function (or by the 
+#' `[loo::loo()]` function with argument `save_psis` set to `TRUE`).
+#' @param ... Currently unused.
+#' @param group A grouping variable (a vector or factor) the same length 
+#' as `y`. Each value in group is interpreted as the group level pertaining 
+#' to the corresponding value of `y`. If `FALSE`, ignored.
+#' @param outlier_thresh Flag values when the difference in the ELPD exceeds
+#' this threshold. Defaults to `NULL`, in which case no values are flagged.
+#' @param size,alpha,jitter Passed to `[ggplot2::geom_point()]` to control 
+#' aesthetics. `size` and `alpha` are passed to to the `size` and `alpha` 
+#' arguments of `[ggplot2::geom_jitter()]` to control the appearance of 
+#' points. `jitter` can be either a number or a vector of numbers.
+#' Passing a single number will jitter variables along the x axis only, while 
+#' passing a vector will jitter along both axes.
+#' @param quantiles Boolean that determines whether to plot the quantiles of
```

Comment:
For me `quantiles` sounds like it would accept vector of quantiles (real valued) as, e.g., dnorm and pnorm. I suggest that `quantiles` parameter would actually allow to define which quantiles are plotted and the type of the plot would be selected with argument called, e.g., `type`. With options type='y' (default) and type='quantiles', which would allow then also extending to other possible types- Can you also include example of how these plots look like in the discussion thread of this PR?

Reply:
> For me `quantiles` sounds like it would accept vector of quantiles (real valued) as, e.g., dnorm and pnorm. 
This does seem like it might be confusing; I think it makes sense to remove this, since users can provide transformations of the values themselves by replacing `y`.

> I suggest that `quantiles` parameter would actually allow to define which quantiles are plotted and the type of the plot would be selected with argument called, e.g., `type`. With options type='y' (default) and type='quantiles', which would allow then also extending to other possible types
I'm not sure what you mean here; could you elaborate?

> Can you also include example of how these plots look like in the discussion thread of this PR?
Will do.

Reply:
If users can provide transformations of the values themselves by replacing y, and this option is not needed then I think removing it as you did is the correct action and you don't need to care about the rest I said.